### ISSUE TITLE
Facebook integration display contributor/autor div

### ISSUE THREAD

**Issue by M5oul** [duniter/blog-posts#91]
‎[01:45:52] ‎Gildas‎: Salut,
‎[01:46:04] ‎Gildas‎: comme ça je ne sais pas si c'est facile à faire ou pas
‎[01:46:10] ‎Gildas‎: j'ai une petite suggestion
‎[01:47:00] ‎Gildas‎: ce serait de changer le texte qui accompagne le lien vers duniter quand je mets le lien vers duniter dans facebook, ça donne ça : www.icarart.com/web/Sans titre-1.jpg
‎[01:47:35] ‎Gildas‎: ce serait pas mal d'avoir un micro texte qui parle de la monnaie libre, plutôt que le truc sur le contributeur.
‎[01:47:36] ‎Gildas‎: merci
‎[01:47:37] ‎Gildas‎: a+

![sans titre-1](https://cloud.githubusercontent.com/assets/4758871/18896527/adbd0f14-8522-11e6-9eef-c9865556144c.jpg)

**Comment by c-geek** [comment]
Oui, Facebook affiche le premier truc qu'il trouve : là il s'agit du bandeau d'auteur. A voir s'il n'existe pas un mécanisme HTML pour mettre une autre valeur.

**Comment by blavenie** [comment]
Il faut utiliser les balises Open Graph (balise meta) : https://developers.facebook.com/docs/sharing/webmasters#markup

**Comment by c-geek** [comment]
OK, il existe dans ghostblog (notre moteur de blog) 2 champs dédiés à cela : 
- title
- description

Par défaut ces champs sont vides, il faudrait donc que notre algorithme de publication remplisse ces 2 champs automatiquement, par extraction du titre et d'une partie du contenu.
